terms.html

```diff
@@ -103,14 +103,15 @@
   <dt><dfn data-lt="issuers|issuer's|issuer(s)|issuing|issued">issuer</dfn></dt>
   <dd>
 A role a <a>party</a> can perform as it constructs a <a>verifiable credential</a> 
-from a set of <a>claims</a> it has <a>authored</a>, meta-data, and proofs,
-typically for the purpose of transferring that to a <a>holder</a>.
+from a set of <a>claims</a> it has <a>authored</a>, including metadata and proofs,
+typically for the purpose of transferring the <a>verifiable credential</a> to 
+a <a>holder</a>.
   </dd>
   <dt><dfn data-lt="party|parties|party's">party</dfn></dt>
-  <dd>
+An <a>entity</a> that has 'a mind of its own', which means, among other things, that it 
 An <a>entity</a> that has 'a mind of its own', which means, amongst other things, that it autonomously (sovereignly)
 <ul>
-  <li>sets the objectives it aims to pursue</li>
+  <li>creates and maintains (<a>authors</a>) data, including <a>identifiers</a>, <a>claims</a>, and <a>verifiable credentials</a></li>
```

Comment:
Seems to be anothing dupication. line 114 is the same as line 115. Did you mean to remove "sets the objectives it aims to pursue" ?